examples/form/dub.json

```
@@ -0,0 +1,12 @@
+{
+	"name": "form_example",
+	"authors": [
+		"Pablo De Napoli"
+	],
+	"dependencies": {
+		"vibe-d": "~>0.8.2"
```

```suggestion
		"vibe-d:web": {"path": "../../"}
```

This line caused some CI builds to get stuck for some reason. It makes sense to to restrict the import to `vibe.web` here (would require to adjust the `import vibe.vibe;`, too), but it should in any case reference vibe.d by path, so that it picks up the version that is actually checked out.

From a pure example point-of-view this is a bit unfortunate, but once I get around to including the examples on the website, I'd make sure to add a simple replacement rule to show version based dependencies there.